packages/engine/src/networking/schema/networkSchema.ts

```diff
@@ -43,6 +52,10 @@ export interface WorldStateInterface {
     leftPose: Pose
     rightPose: Pose
   }[]
+  velocities: {
```

Comment:
velocities should be in the same object as the pose for the same networkId, no need to send networkIds twice and is clearer to have all of an object's information together